Dockerfile

```diff
@@ -62,3 +62,7 @@ RUN make \
 ENV LC_ALL=C.UTF-8
 
 CMD bash
+
+# Local Variables:
```

Comment:
maybe we need to standardize replit with something like https://editorconfig.org/. If you have experience with this or something else lemme know.

Reply:
I've played with it in the past. It seems like the right solution to me, but due to overhead I would wait longer before we implement something like this. (I.e. once it starts to feel like a burden to configure your editor to follow the style.)